.changeset/forty-rats-jog.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": patch
+---
+
+Spinner component size get's large when passing sx and size props
```

Comment:
I think typo here for gets:

```suggestion
Spinner component size gets large when passing sx and size props
```

An alternative for the changelog too, not sure if it fully captures what is being changed though:

```suggestion
Update Spinner component to correctly use the `size` prop when both `sx` and `size` are provided
```